ledger/src/blockstore_meta.rs

```diff
@@ -11,6 +12,35 @@ use {
     },
 };
 
+bitflags! {
+    #[derive(Default, Serialize, Deserialize)]
```

Comment:
let's get rid of `Default` and instead use [`empty()`](https://docs.rs/bitflags/latest/bitflags/#methods)(sry can't directly link 'cause dumb docs) for better self-documentation. it's not necessarily intuitive that the default is empty

Reply:
Fair; I removed the derive and added explicit implementation of `Default` trait to use `empty()` since `SlotMeta` implements `Default` and thus requires all members to also implement the trait

Reply:
RIP. `Default` is the root of all evil